src/main/java/task/Task.java

```diff
@@ -0,0 +1,55 @@
+package task;
+
+import java.io.Serializable;
+
+public class Task implements Serializable {
+    private String name;
+    private boolean isDone;
+
+    /**
+     * The constructor for the Task class
+     *
+     * @param name The name of the task
+     */
+    public Task(String name) {
+        this.name = name;
+        this.isDone = false;
```

Comment:
```suggestion
        isDone = false;
```
Should "this" be here?